index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>HTML Homework</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <style>
+        body {
+            margin: 0;
+            color: #ffeeaa;
+            font-weight: 700;
+            line-height: 2;
+            font-size: calc(14px + (30 - 14) * ((100vw - 300px) / (1600 - 300)));
+        }
+        .grid-container {
+            display: grid;
+            grid-template-areas: 'l l n n n n n n n n n n' 'a a m m m m m m m m m m' 'f f f f f f f f f f f f';
+            grid-gap: 5px;
+            background-color: #ffeeaa;
+            padding: 20px;
+        }
+        .logo {
+            grid-area: l;
+            background: #92cab3;
+        }
+        .nav {
+            grid-area: n;
+            background: #ff706a;
+            display: flex;
+            justify-content: space-between;
```

Comment:
"justify-content" property doesn't have sense here. You can remove it without any impact